rollup.config.js

```diff
@@ -34,7 +34,7 @@ export default [{
 	output: {
 		file: 'dist/vis-timeline-graph2d.min.js',
 		name: 'vis',
-		exports: 'named',
```

Comment:
I think we need `named` here!

Reply:
Nope. We changed the `index.js` to `export default` right here:
https://github.com/visjs/vis-timeline/pull/5/files#diff-168726dbe96b3ce427e7fedce31bb0bcR43